File: 백준/Bronze/3029. 경고/경고.py
import sys
input = sys.stdin.readline

start_h, start_m, start_s = map(int, input().rstrip().split(':'))
end_h, end_m, end_s = map(int, input().rstrip().split(':'))

time = (end_h*3600+end_m*60+end_s) - (start_h*3600+start_m*60+start_s)
if time <= 0: time += (24*3600)
print('%02d:%02d:%02d'%(time//3600, (time%3600)//60, time%60))

File: 백준/Bronze/3029. 경고/경고.cc
#include <stdio.h>

int main(void) {
    int start_h, start_m, start_s, end_h, end_m, end_s;
    scanf("%d:%d:%d", &start_h, &start_m, &start_s);
    scanf("%d:%d:%d", &end_h, &end_m, &end_s);

    start_s += (start_m) * 60 + (start_h) * 3600;
    end_s += (end_m) * 60 + (end_h) * 3600;
    int result = end_s - start_s;

    if (result <= 0) result += 86400;
    printf("%02d:%02d:%02d", result/3600, result%3600/60, result%60);
    return 0;
}